frontend/src/components/AdminViewRoom/SeatPlacer.tsx

```diff
@@ -0,0 +1,81 @@
+import KnappBase from 'nav-frontend-knapper';
+import PanelBase, { Panel } from 'nav-frontend-paneler';
+import { Select } from 'nav-frontend-skjema';
+import React from 'react';
+import { IUser } from '../../API/interfaces';
+import { ISeat } from '../ViewRooms';
+
+interface IProps {
+  seat: ISeat;
+  users?: IUser[];
+  assign: (user: IUser, seat: ISeat) => void;
+  delete: (seat: ISeat) => void;
+}
+
+interface IState {
+  selectedUser?: IUser;
+}
+
+class SeatPlacer extends React.Component<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+    this.state = {
+      selectedUser: undefined,
+    };
+  }
+  public render() {
+    const { seat, users } = this.props;
+    const { selectedUser } = this.state;
+
+    if (seat.user) {
+      return (
+        <Panel border={true} style={{ display:'flex', width:'35%' }}>
+          <PanelBase border={true}>{seat.id}</PanelBase>
+          <PanelBase border={true}>{seat.user.username}</PanelBase>
+          <KnappBase type="fare" onClick={this.delete}>X</KnappBase>
+        </Panel>
+      );
+    }
+    if (!users) return null;
+    const userOptions = this.usersToOption(users);
+    return (
+      <Panel border={true} style={{ display:'flex', width:'35%' }}>
+        <PanelBase border={true}>{seat.id}</PanelBase>
+        <Select label="" onChangeCapture={this.selectUser}>
+          <option value="">Select user</option>
+          {userOptions}
+        </Select>
+        <KnappBase type="hoved" disabled={selectedUser ? false : true} onClick={this.assign}>
+          Assign Seat
```

Comment:
This could be stringed like _EDIT_ROOM_BUTTON, but not necessary

Reply:
You're right! I'll fix it